eos/scattering/observables.cc

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright (c) 2021 Méril Reboud
+ *
+ * This file is part of the EOS project. EOS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License version 2, as published by the Free Software Foundation.
+ *
+ * EOS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#include <eos/observable-impl.hh>
+#include <eos/scattering/ee-to-ccbar.hh>
+#include <eos/utils/concrete_observable.hh>
+#include <eos/utils/concrete-cacheable-observable.hh>
+
+namespace eos
+{
+    // ee -> ccbar
+    // {{{
+    ObservableGroup
+    make_ee_to_ccbar_group()
+    {
+        auto imp = new Implementation<ObservableGroup>(
+            R"(Observables in $ee \to c\bar{c}$ processes)",
+            R"(The options "nchannel" and "nresonance" fix the number of channels and resonances respectively.)",
```

Comment:
Is this comment still true? I do not see how.

Reply:
No, this is something we planned at the beginning. I removed the comment.